Gruntfile.js

```diff
@@ -196,36 +203,39 @@ module.exports = function( grunt ) {
 
 	grunt.registerTask( "lint", [
 		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		runIfNewNode( "eslint" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
+	grunt.registerTask( "lint:newer", [
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint" )
+	] );
 
-	grunt.registerTask( "test", [ "test_fast" ].concat(
-		[ runIfNewNode( "promises_aplus_tests" ) ]
-	) );
+	grunt.registerTask( "test:fast", runIfNewNode( "node_smoke_tests" ) );
+	grunt.registerTask( "test:slow", runIfNewNode( "promises_aplus_tests" ) );
 
-	// Short list as a high frequency watch task
-	grunt.registerTask( "dev", [
-			"build:*:*",
-			runIfNewNode( "newer:eslint:dev" ),
-			"uglify",
-			"remove_map_comment",
-			"dist:*"
-		]
-	);
+	grunt.registerTask( "test", [
+		"test:fast",
+		"test:slow"
+	] );
 
-	grunt.registerTask( "default", [
-		"dev",
-		runIfNewNode( "eslint:dist" ),
-		"test_fast",
+	grunt.registerTask( "dev", [
+		"build:*:*",
+		"newer:eslint:dev",
```

Comment:
This particular mistake wouldn't be caught by Travis as the `dev` task is not invoked there. Fortunately the mistake was repeated in more places. :-)